docs/data/gridftp.md

```diff
@@ -111,6 +111,27 @@ All other configuration settings should be suitable for most purposes. However,
 
 ### Optional configuration
 
+#### Setting transfer limits
+
+Create file named `/etc/sysconfig/gridftp-limits`. To set a limit on the total or per-user number of transfers, set the following configuration:
+
+    :::file
+    export GRIDFTP_TRANSFER_LIMIT="80"
+    export GRIDFTP_DEFAULT_USER_TRANSFER_LIMIT="50"
+    export GRIDFTP_LIGO_USER_TRANSFER_LIMIT="40"
+    export GRIDFTP_CMS_USER_TRANSFER_LIMIT="20"
+
+In the above configuration:
+
+- There would be no more than 80 transfers going at a time, across all users.
+- By default, any single user can have no more than 50 transfers at a time.
+- The `ligo` user has a more stringent limit of 40 transfers at a time.
+- The `cms` user has an even more stringent limit of 20 transfers at a time.
```

Comment:
Rather than another example, let's be more explicit that you can set limits for a specific user with `GRIDFTP_<UNIX USERNAME>_USER_TRANSFER_LIMIT`, where `<UNIX USERNAME>` is the unix username in all caps.